### PR TITLE
Fetch data from api and display it in a table

### PR DESCRIPTION
**Pull request by OlenaKashuba**
- Create a new component AdminGeneral, folder for it and child component
- Create a table for displaying data
- Do fetch  of applicants from api
- Display applicants data into a table
- Add style files to AdminGeneral and ApplicantDashboard
- Install and enable 'cors' module to fix the Cross-Origin Resource Sharing issues.

### CODE REVIEW

File: backend/fakeData/generate.js [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+    tel: '{{ phone() }}',  
+    status: '{{ bool() }}',      
+    country: faker.address.country,
+    experience: '{{ company().toUpperCase() }}',
+    itAccess: '{{ bool() }}',
+    hearAbout: faker.random.word
+  }
+];
+ 
+madoka.save(template, path.join(__dirname, 'fakeApplicant.json'));
```
Comment (dahfool) [inline backend/fakeData/generate.js]: @OlenaKashuba What is madoka and we really need it ?

File: frontend/src/Components/AdminGeneral/AdminGeneral.js [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+
+class AdminGeneral extends React.Component {
+	constructor(props) {
+		super(props);
+
+		this.state = {
+			applicants: []
+		}
+
+		this.getListofApplicants = this.getListofApplicants.bind(this);
```
Comment (LouiaR) [inline frontend/src/Components/AdminGeneral/AdminGeneral.js]: @OlenaKashuba  you can avoid writing  this (line 12)  by using arrow function

Comment (LouiaR) [reply frontend/src/Components/AdminGeneral/AdminGeneral.js]: I think it will be great to keep everything concerning the admin within the same folder

Comment (OlenaKashuba) [reply frontend/src/Components/AdminGeneral/AdminGeneral.js]: @LouiaR i think I cannot make this component stateless cause it is using ComponentDidMount

@LouiaR Relating to admin component, i don't think it is necessary, but I can add another Admin folder on top of both of admin components. Will that help?

Comment (LouiaR) [reply frontend/src/Components/AdminGeneral/AdminGeneral.js]: Yes, thank you.

**Pull request merged**